eng/common/dotnet-install.sh

```diff
@@ -70,7 +70,7 @@ case $cpuname in
     ;;
 esac
 
-dotnetRoot="$repo_root/.dotnet"
+dotnetRoot="${repo_root}.dotnet"
 if [[ $architecture != "" ]] && [[ $architecture != $buildarch ]]; then
   dotnetRoot="$dotnetRoot/$architecture"
```

Comment:
>"$dotnetRoot/$architecture" [](http://example.com/codeflow?start=13&length=27)

worth it to use join-path?

Reply:
Everything in eng/common is just copied from arcade by darc.  We don't need to review it -- or reviews should be done in dotnet/arcade before it flows to consumers.